src/runtime/button-api.js

```diff
@@ -51,6 +52,20 @@ const TITLE_LANG_MAP = {
 };
 
 
+/**
+   * @param {!AnalyticsEvent} analyticsEvent
+   * @return {!../api/client-event-manager-api.ClientEvent}
+   */
+export function CreateButtonRelatedClientEvent(analyticsEvent) {
```

Comment:
Seems like a useful helper function and not actually related to a button at all.  Perhaps this should be called createSwgClientEvent(eventType, isFromUserAction)

With isFromUserAction optional and defaulting to false.

I'd also move it to a more centralized location.